src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1684,14 +1684,25 @@ public static function unsetConnectionResolver()
         static::$resolver = null;
     }
 
+    /**
+     * Get the default table name associated with the model.
+     *
+     * @param self|null $class
```

Comment:
this phpdoc is wrong. this should say `class-string<self>|null` or just `string|null`.

Reply:
Thank you. I modified it